includes/class-scd-ext-lesson-frontend.php

```diff
@@ -97,6 +100,23 @@ public function lesson_content_drip_filter( $lessons ) {
 		return $lessons;
 	}
 
+	/**
+	 * Hides the comments if the lesson is blocked by content drip.
+	 *
+	 * @since  2.3.0
+	 * @access private
+	 */
+	public function hide_comments() {
+
+		if ( 'lesson' !== get_post_type() ) {
+			return;
+		}
+
+		if ( Sensei_Content_Drip::instance()->access_control->is_lesson_access_blocked( get_the_ID() ) ) {
+			remove_action( 'sensei_pagination', array( 'Sensei_Lesson', 'output_comments' ), 90 );
```

Comment:
I believe we're trying to use short array syntax now (i.e. `[]` vs. `array()`).

Reply:
It is advised to use the long array syntax in the [coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#declaring-arrays).

Reply:
@alexsanford @jom What's your take on this?

Reply:
In WP core, this was [going to change](https://make.wordpress.org/core/2019/03/26/coding-standards-updates-for-php-5-6/), but then [didn't](https://make.wordpress.org/core/2019/07/12/php-coding-standards-changes/) (a bit controversially from the comments). Internally, it seems like there other other new projects, such as [Newspack](https://github.com/Automattic/newspack-plugin/), using short-array syntax. [WPCS](https://github.com/WordPress/WordPress-Coding-Standards) has left long/short-array rules out entirely, which is generally what we're trying to stick to. I'm definitely on team short-array because it is what modern PHP developers outside of WordPress have been using for years.

However, for these sorts of things, I'd rather change it project-wide (like WCPC, WPJM, Resumes, and Applications) instead of just in parts of new code. I think consistency within the file is more important for readability.